src/Gallery/Movies-get.js

```diff
@@ -0,0 +1,34 @@
+import Gladiator from './Gladiator.jpg';
+import LOTR from './LOTR.jpg';
+import PulpFiction from './Pulp Fiction.jpg';
+import StarWars from './Star Wars.jpg';
+import Terminator from './Terminator.jpg';
+import TopGun from './Top Gun.jpg';
+
+export default function getMovies() {
```

Comment:
Could you please work a bit on the indentation and break line strategy?

```
[{ 
    name: "Gladiator", 
    picture: Gladiator, 
    id: 'Galdiator'
}, { 
    name: "Lord of The Ring", 
    picture: LOTR, 
    id: 'LordofTheRings' 
},
...
```

Reply:
done.